### ISSUE TITLE
Don't show "Downloads" tab under "My Account" if there are no digital products

### ISSUE THREAD

**Issue by jnachtigall** [woothemes/woocommerce#11478]
If a shop does not sell any virtual/digital products, there is never going to be any "Downloads". I think the "Downloads" tab should only be shown if there are any appropriate products. Otherwise it's confusing for customers.

(Anyway, I know this can be easily fixed with for now
```
.woocommerce-MyAccount-navigation-link--downloads { 
  display: none; 
}
```
but I guess this would make sense by default.)

**Comment by claudiosmweb** [comment]
No need any extra code, since you can disable:

![screenshot from 2016-07-19 19-22-01](https://cloud.githubusercontent.com/assets/1264099/16968577/189f01a8-4de6-11e6-8b88-816ffff16c6e.png)

**Comment by jnachtigall** [comment]
@claudiosmweb Thanks. I was deliberately looking onto this settings page before creating the issue. And did not see this :-/ In fact, I did even hover the (?) next to the "Downloads" input field. I would propose, from a UX point, that the info "Leave blank to disable this endpoint" is added to the (?) text.

**Comment by claudiosmweb** [comment]
@jnachtigall this is new, since 2.6.2.

**Comment by jnachtigall** [comment]
@claudiosmweb Nope, even in 2.6.4 it is `'Endpoint for the My Account &rarr; Downloads page'`. I'd suggest to make this `'Endpoint for the My Account &rarr; Downloads page. Leave blank to disable this endpoint`'.

See https://github.com/woothemes/woocommerce/blob/7701d4ce99175a1d7c6617feb5071c7b88cbe0a8/includes/admin/settings/class-wc-settings-accounts.php#L132

It's just more useful to have this info next to where you enter stuff. Having it just at top means it's easy to miss it (like I did). I know this is a minor UX nitpicking thing.

**Comment by claudiosmweb** [comment]
@jnachtigall You don't understand what do I said, I said that this is new, which was implemented in 2.6.2.
